src/sections/richtext.liquid

```diff
@@ -0,0 +1,118 @@
+<section class="richtext {% if section.settings.background_color_preset != 'none' and section.settings.background_color_preset != blank %}{{ section.settings.background_color_preset | prepend: 'section-color-preset__' }}{% endif %}" data-preview-item="richtext">
+  <div class="richtext__container">
+    <div class="richtext__wrapper">
+      {% if section.settings.heading != blank %}
+        <header class="section__headings">
+          <h3 class="section__heading">{{ section.settings.heading }}</h3>
+          {% if section.settings.subheading != blank %}
+            <p class="section__subheading">{{ section.settings.subheading }}</p>
+          {% endif %}
+        </header>
+      {% endif %}
+      <article class="section__content">
+        <section class="section__body">
+          {{ section.settings.body }}
+        </section>
+      </article>
+    </div>
+  </div>
+</section>
+
+{% style %}
+
+section.richtext {
```

Comment:
overspecified, why not just `.richtext`?

Reply:
It's more of a convention I have used throughout all of the templates and comes into play particularly for anything that has style configurations. Wanted to give it a +1 to its specificity power.